src/spikeinterface/core/waveforms_extractor_backwards_compatibility.py

```diff
@@ -361,6 +368,8 @@ def _read_old_waveforms_extractor_binary(folder):
     with open(params_file, "r") as f:
         params = json.load(f)
 
+    return_scaled = (params["return_scaled"],)
```

Comment:
Why not just 
```suggestion
    return_scaled = params["return_scaled"]
```
???